design-patterns/domain-driven-design/README.md

```diff
@@ -0,0 +1,20 @@
+# Domain Driven Design
+
+A software developer named Eric Evans recognized the need to understand the software domain. To introduce the DDD approach, he wrote a book called Domain-Driven Design: Tackling Complexity in the Heart of Software. In his book, Evans says, “The heart of the software is its ability to solve domain-related problems for its user. All other features, vital though they may be, support this basic purpose.”
+
+The most obvious advantage of DDD is that it gets everybody using the same language. When development teams use the same language as domain experts, it leads to software design that makes sense to the end user. Because the terminology in the application corresponds to real-world activities, there is less confusion and users will understand how to use the product more quickly.
+
+Check out four DDD patterns that we apply to the Vendia Share Univseral Application( we call it `Uni`).
+
+## Pattern 1: Single Domain, Single Bounded Context, and a Single Uni
```

Comment:
I'd add links to each of the sub directories from these Pattern Overview sections

Reply:
accepted